packages/backend/src/graphql/resolvers/liquidity.test.ts

```diff
@@ -104,6 +104,7 @@ describe('Liquidity Resolvers', (): void => {
     })
 
     test('Returns an error for invalid id', async (): Promise<void> => {
+      let error
```

Comment:
in this file, because we had expects in `beforeEach`/`beforeAll` functions, doing `expect.assertions` wasn't intuitive at all (sometimes # of actual assertions would end up being 2 or 3 more than in the test block itself), so I just decided to check for the errors this way